docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -0,0 +1,252 @@
+In this example, we'll port-forward the [Event Publisher Proxy](../../05-technical-reference/00-architecture/evnt-01-architecture.md) to localhost.
+
+1. Port-forward the Event Publisher Proxy Service to localhost. We will use port `3000`. Run:
+   ```bash
+   kubectl -n kyma-system port-forward service/eventing-event-publisher-proxy 3000:80
+   ```
+2. Now publish an event to trigger your function. In another Terminal window run:
+
+   <div tabs name="Publish an event" group="trigger-workload">
+     <details open>
+     <summary label="CloudEvents Conformance Tool">
+     CloudEvents Conformance Tool
+     </summary>
+   
+      ```bash
+      cloudevents send http://localhost:3000/publish \
+         --type sap.kyma.custom.myapp.order.payment-success.v1 \
+         --id e4bcc616-c3a9-4840-9321-763aa23851fc \
+         --source myapp \
+         --datacontenttype application/json \
+         --data "{\"orderCode\":\"3211213\", \"orderAmount\":\"1250\"}" \
+         --yaml
+      ```
+   
+     </details>
+     <details>
+     <summary label="curl">
+     curl
+     </summary>
+   
+      ```bash
+      curl -v -X POST \
+           -H "ce-specversion: 1.0" \
+           -H "ce-type: sap.kyma.custom.myapp.order.payment-success.v1" \
+           -H "ce-source: myapp" \
+           -H "ce-eventtypeversion: v1" \
+           -H "ce-id: e4bcc616-c3a9-4840-9321-763aa23851fc" \
+           -H "content-type: application/json" \
+           -d "{\"orderCode\":\"3211213\", \"orderAmount\":\"1250\"}" \
+           http://localhost:3000/publish
+      ```
+     </details>
+   </div>
+
+## Verify the event delivery
+
+To verify that the event was properly delivered, check the logs of the Function:
+
+<div tabs name="Verify the event delivery" group="trigger-workload">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In Kyma Dashboard, return to the view of your `lastorder` Function.
+2. Go to **Code** and find the **Replicas of the Function** section.
+3. Click on **View Logs**.
+
+</details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+Run: 
+
+```bash
+kubectl logs -f -n default \
+  $(kubectl get pod \
+    --field-selector=status.phase==Running \
+    -l serverless.kyma-project.io/function-name=lastorder \
+    -o jsonpath="{.items[0].metadata.name}")
+```
+
+  </details>
+</div>
+
+You will see the received event in the logs:
+```
+Received event:  { orderCode: '3211213', orderAmount: '1250' } , Event Type:  sap.kyma.custom.myapp.order.paymentsuccess.v1
+```
+Note that the `Event Type` of the received event is not the same as defined in the Subscription.
+
+## Conclusion
+
+We saw that Eventing services modifies the event names to filter out non-alphanumeric character to conform to Cloud Event specifications. But this modification is abstract to the user because the user can still publish and subscribe to the original Event names. 
```

Comment:
```suggestion
You see that Eventing services modify the event names to filter out non-alphanumeric character to conform to Cloud Event specifications. 

> **CAUTION:** This cleanup modification is abstract; you can still publish and subscribe to the original Event names. However, in some cases, it can lead to a naming collision as explained in [Event names](../../05-technical-reference/evnt-01-event-names.md).
```
We've been addressing the reader/user in all the tutorial so far, and now suddenly we talk about the users in third person as if explaining the architecture to a dev colleague? This feels weird. I tried rephrasing it as informative box, combined with the following sentence.